### PR TITLE
Feat: upgrade version to 1.2.3

### PR DESCRIPTION
**Pull request by barnettZQG**
Signed-off-by: barnettZQG <barnett.zqg@gmail.com>

### CODE REVIEW

**Review by wonderflow — CHANGES_REQUESTED** [review]
hold until we release

File: i18n/zh/docusaurus-plugin-content-docs/current/install.mdx [hunk 1]
```diff
@@ -260,6 +202,14 @@ sudo mv ./vela /usr/local/bin/vela
 </TabItem>
 </Tabs>
 
+## 3. 安装 KubeVela Core
+
+> vela CLI 版本需要 >=1.2.3
+
+```shell script
+vela install
+```
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/install.mdx:211]: We're should print the expected output

**Review by wonderflow — APPROVED** [review]

File: docs/install.mdx [hunk 2]
```diff
@@ -255,6 +206,25 @@ sudo mv ./vela /usr/local/bin/vela
...
 
+## 2. Install KubeVela Core
+
+> The version of vela CLI >= 1.2.3.
+
+```shell script
+vela install
+```
+
+<details> <summary> check out the outcome </summary>
```
Comment (wonderflow) [inline docs/install.mdx:217]: 要把结果贴出来

Comment (barnettZQG) [reply docs/install.mdx:217]: 贴出来了呀。

**Pull request merged**